src/settings-ui/Settings.UI/SettingsXAML/Views/DashboardPage.xaml

```diff
@@ -348,17 +348,28 @@
                                             <ColumnDefinition Width="Auto" />
                                             <ColumnDefinition Width="Auto" />
                                         </Grid.ColumnDefinitions>
-                                        <Image Width="20" Margin="0,0,0,0">
+                                        <Image
+                                            Grid.Column="0"
+                                            Width="20"
+                                            Margin="0,0,0,0">
                                             <Image.Source>
                                                 <BitmapImage UriSource="{x:Bind Icon, Mode=OneWay}" />
                                             </Image.Source>
                                         </Image>
-                                        <TextBlock
+                                        <StackPanel
                                             Grid.Column="1"
                                             VerticalAlignment="Center"
-                                            FontWeight="SemiBold"
-                                            Text="{x:Bind Label, Mode=OneWay}"
-                                            TextTrimming="CharacterEllipsis" />
+                                            Orientation="Horizontal">
+                                            <TextBlock
+                                                VerticalAlignment="Center"
+                                                FontWeight="SemiBold"
+                                                Text="{x:Bind Label, Mode=OneWay}"
+                                                TextTrimming="CharacterEllipsis" />
+                                            <InfoBadge
+                                                Margin="4,0,0,0"
+                                                Style="{StaticResource NewInfoBadge}"
+                                                Visibility="{x:Bind IsNew, Converter={StaticResource BoolToVisibilityConverter}, Mode=OneWay}" />
```

Comment:
BoolToVisibilityConverter is embedded, so may be omitted :)

Reply:
I think we should prefer as explicit as possible :) Makes intent easier to read when going through the code if one doesn't know the defaults.